src/main/java/uk/gov/hmcts/reform/civil/service/PaymentRequestUpdateCallbackService.java

```diff
@@ -46,44 +52,59 @@ public void processCallback(ServiceRequestUpdateDto serviceRequestUpdateDto) {
             CaseDetails caseDetails = coreCaseDataService.getCase(Long.valueOf(serviceRequestUpdateDto
                                                                                    .getCcdCaseNumber()));
             CaseData caseData = caseDetailsConverter.toCaseData(caseDetails);
-            caseData = updateCaseDataWithStateAndPaymentDetails(serviceRequestUpdateDto, caseData);
-            createEvent(caseData, SERVICE_REQUEST_RECEIVED,
-                        serviceRequestUpdateDto.getCcdCaseNumber()
-            );
+            caseData = updateCaseDataWithStateAndPaymentDetails(serviceRequestUpdateDto, caseData, feeType);
+            if (feeType.equals(FeeType.HEARING.name()) || feeType.equals(FeeType.CLAIMISSUED.name())) {
+                createEvent(caseData, serviceRequestUpdateDto.getCcdCaseNumber(), feeType);
+            }
 
         } else {
-
-            log.error("Case id {} not present", serviceRequestUpdateDto.getCcdCaseNumber());
+            log.error("Service request status is not PAID for Case id {}",
+                      serviceRequestUpdateDto.getCcdCaseNumber());
         }
     }
 
-    private void createEvent(CaseData caseData, CaseEvent eventName, String generalApplicationCaseId) {
+    private void createEvent(CaseData caseData, String caseId, String feeType) {
 
         StartEventResponse startEventResponse = coreCaseDataService.startUpdate(
-            generalApplicationCaseId,
-            eventName
+            caseId,
+            getEventNameFromFeeType(caseData, feeType)
         );
+
         CaseData startEventData = caseDetailsConverter.toCaseData(startEventResponse.getCaseDetails());
-        BusinessProcess businessProcess = startEventData.getBusinessProcess()
-            .updateActivityId(serviceRequestReceived);
+        BusinessProcess businessProcess = null;
+        if (feeType.equals(FeeType.HEARING.name())) {
+            businessProcess = startEventData.getBusinessProcess()
+                .updateActivityId(serviceRequestReceived);
+        }
 
         CaseDataContent caseDataContent = buildCaseDataContent(
             startEventResponse,
             caseData,
             businessProcess
         );
-        data = coreCaseDataService.submitUpdate(generalApplicationCaseId, caseDataContent);
-        coreCaseDataService.triggerEvent(caseData.getCcdCaseReference(), SERVICE_REQUEST_RECEIVED);
 
+        coreCaseDataService.submitUpdate(caseId, caseDataContent);
+    }
+
+    private CaseEvent getEventNameFromFeeType(CaseData caseData, String feeType) {
+        if (feeType.equals(FeeType.HEARING.name())) {
+            return SERVICE_REQUEST_RECEIVED;
+        } else if (feeType.equals(FeeType.CLAIMISSUED.name())
+                   && isSpecCaseCategory(caseData, featureToggleService.isAccessProfilesEnabled() ? true : false)) {
```

Comment:
`? true : false` is being applied to a boolean. There's no need for this. You can just use `featureToggleService.isAccessProfileEnabled()`.

Reply:
Yes, this was superfluous, removed it - resolved myself, as you are still off on sick leave.